modules/storage-persistent-storage-volume-encrypt-with-kms-key.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage-aws.adoc
+
+[id="aws-container-persistent-volumes-encrypt_{context}"]
+= Encrypting container persistent volumes on AWS with a specific KMS key
+
+.Prerequisites
+
+* Storage must exist in the underlying infrastructure before you encrypt the EBS instance volumes in {product-title}.
+* You must create a customer KMS key on AWS before you encrypt the container-persistent volumes in {product-title}.
```

Comment:
`before you encrypt the container-persistent volumes in {product-title}.`
Redundant sentence like above

Reply:
Addressed